py/create_mnar.py

```diff
@@ -0,0 +1,47 @@
+"""
+Create missing not at random (MNAR) data
+"""
+import pandas as pd
+import numpy as np
+
+def create_mnar(df, a0, a1, share_yes, share_no):
+  """
+  df: A pandas data frame
+  a0: the intercept for the G2 missingness probability
+  a1: the slope for the G2 missingness probability
+  share_yes: the probability of switching a yes to na
+  share_no: the probability of switching a no to na
+  """
+  
+  def g3_to_na(x, a0, a1):
+    """
+    Rendomly turn values from g3 to NaN
+    Different rates can be applied with the intercept a0 and age coefficient a1
+    """
+    rand = np.random.uniform(size=1)
+    prob = a0 + a1 * x
```

Comment:
Shouldn't the probability be something like:

`exp[a0 + a1*x]/(1 + exp[a0 + a1x]` 

?

Based on this: https://github.com/lbotti319/bayesian_math_scores/issues/7#issuecomment-835535744

Reply:
Just because our model is sigmoidal doesn't mean the underlying mechanism needs to be. We could use linear probability for one and logistic for the other. Either way, I made the switch.